content/docs/aws/authentication-oidc.md

```diff
@@ -0,0 +1,129 @@
+
+In order to authenticate and manage users for Kubeflow, let's first connect Auth0 app to Github. You can follow instructions [here](https://auth0.com/docs/connections/social/github).
+
+The details of instruction are listed below:
+
+### 1. Add a new Application in Github
+To add a new application, log in to [GitHub](https://github.com/) and go to **OAuth Apps** in your [developer settings](https://github.com/settings/developers). Next click [Register a new application](https://github.com/settings/applications/new).
+
+<img src="/docs/images/aws/github-oauth-app-register.png"
+  alt="Github Oauth App Register"
+  class="mt-3 mb-3 border border-info rounded">
+
+### 2. Register Your New App
+On the [Register a new application](https://github.com/settings/applications/new) page fill out the form with the following information. Modify the parameters to reflect your application.
+
+If your Auth0 domain name is not shown above and you are not using our custom domains feature, your domain name is your tenant name, plus `.auth0.com`. For example, if your tenant name were `kftest`, your Auth0 domain name would be `kftest.auth0.com` and your redirect URI would be `https://kftest.auth0.com/login/callback`.
+
+<img src="/docs/images/aws/github-oauth-app-register-details.png"
+  alt="Github Oauth App Register Details"
+  class="mt-3 mb-3 border border-info rounded">
+
+After completing the form click **Register application** to proceed.
+
+### 3. Get your GitHub app's Client ID and Client Secret
+Once the application is registered, your app's `Client ID` and `Client Secret` will be displayed on the following page:
+
+<img src="/docs/images/aws/github-oauth-app-client-id-secret.png"
+  alt="Github Oauth App Client ID Secret"
+  class="mt-3 mb-3 border border-info rounded">
+  
+### 4. Copy your GitHub app's Client ID and Client Secret
+
+Go to your [Auth0 Dashboard](https://manage.auth0.com/dashboard/), if you haven't sign in with Auth0 yet, you can login at first.
+
+<img src="/docs/images/aws/auth0-login.png"
+  alt="Auth0 Login"
+  class="mt-3 mb-3 border border-info rounded">
+ 
+Then you can look at below page and modify the domain name Under `TENANT DOMAIN` to be the domain name you set up in Step 2.
+
+<img src="/docs/images/aws/auth0-welcome-page.png"
+  alt="Auth0 Welcome Page"
+  class="mt-3 mb-3 border border-info rounded">
+ 
+After modification, click **NEXT** button. Then you can set up your personal information there and click **CREATE ACCOUNT** button.
+ 
+After log into Auth0 Dashboard, select **Connections > Social**, then choose Github. Copy the `Client ID` and `Client Secret` from the `Developer Applications` of your app on Github into the fields on this page on Auth0.
+
```

Comment:
Can you adjust the sequence? it would be better to register auth0 because tenant name may not be available.

Reply:
No problem! Will fix it now